### ISSUE TITLE
Recompile for 64 bits (HL25-anniversary)

### ISSUE THREAD

**Issue by denz8028** [YaLTeR/BunnymodXT#553]
`ERROR: ld.so: object '/home/den4ik/gimno/libBunnymodXT.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.`

Then game crashes.

**Comment by YaLTeR** [comment]
It won't help. The entire hooking needs to be redone if we want to support the 25th anniversary builds on Linux. Which I personally don't think is worth the effort.

**Comment by khanghugo** [comment]
I agree that there should be linux instruction. In the meantime, if you don't know how to get BXT working with linux, what you have to to is using `runhl.sh` script in bxt-rs repository. There should also be disclaimer about BXT not working for HL25 as well.

**Comment by SmileyAG** [comment]
- Valve stopped exporting all dynamic symbols to the `.dynsym` section starting with the 25th anniversary update, which is why they cannot be picked up with `dlsym` now. 
- There are no implementation as to find addresses of functions or global variables by using DWARF info in [SPTLib](https://github.com/YaLTeR/SPTLib), we can always consider as temp workaround to manually find patterns for functions and offsets to global variables, but that definitely not worth it at all.
- GoldSrc is still 32-bit. Don't be fooled.

For now the only available option that the normal user can do now is rollback to older versions by select the `steam_legacy` branch in the beta versions of game in Steam client:

![Image](https://github.com/user-attachments/assets/50d82878-dbf9-4d9c-9ec5-3c54c0aadfce)

Perhaps, that should be mentioned in README or some other dedicated documentation for that. See that later.

Closing that issue here, I think it better to be opened in SPTLib with given solutions, who knows may someone pick up to do it.
